File: cp_nim/src/bin/abc426-b-the-odd-one-out.nim
# Created by Ayush Biswas at 2026/07/18 18:28
# https://atcoder.jp/contests/adt_easy_20260618_1/tasks/abc426_b
import ../lib/cpio
import ../lib/itertools

# @code begin

import sugar

var r = newReader()

let s = r[string].groupBy(k => k)

for k, v in s:
  if v.len == 1:
    echo k
    break

# @code end

File: cp_rust/src/bin/abc426-b-the-odd-one-out.rs
// Created by Ayush Biswas at 2026/07/07 13:07
// https://atcoder.jp/contests/adt_easy_20260618_1/tasks/abc426_b
use cp_lib::*;

// @code begin
use cpio::*;
use itertools::Itertools;

sol! {
    fn solution(
        s: String
    ) -> char {
        let c = s
            .chars()
            .sorted()
            .group_by(|&x| x)
            .filter(|g| g.len() == 1)
            .next()
            .unwrap();
        c[0]
    }
}
// @code end
